tree/ntuple/v7/src/RMiniFile.cxx

```diff
@@ -418,6 +420,27 @@ struct RTFObject {
    explicit RTFObject(std::uint32_t bits) : fBits(bits) {}
 };
 
+/// Streamer info for data member RNTuple::fChecksum
+struct RTFStreamerElementChecksum {
+   RUInt32BE fByteCount{0x40000000 | (sizeof(RTFStreamerElementChecksum) - sizeof(RUInt32BE))};
+   RUInt16BE fVersion{4};
+
+   RUInt32BE fByteCountNamed{0x40000000 | (sizeof(RUInt16BE) + sizeof(RTFObject) + 11)};
+   RUInt16BE fVersionNamed{1};
+   RTFObject fObjectNamed{0x02000000 | 0x01000000};
```

Comment:
I trust you and @pcanal here (and below) :smile:; the meaning of these bitmasks is opaque to me.